src/main/groovy/no/skatteetaten/aurora/gradle/plugins/AuroraPlugin.groovy

```diff
@@ -44,14 +46,14 @@ class AuroraPlugin implements Plugin<Project> {
 
       mavenTools.setDefaultTasks()
 
-      if (config.applyJavaDefaults.toBoolean()) {
-        reports.add(java.applyJavaDefaults(config.javaSourceCompatibility))
-      }
-
       if (config.applyDefaultPlugins.toBoolean()) {
         reports.add(java.applyDefaultPlugins())
       }
 
```

Comment:
This order was changed because a project that does not explicitly import the java plugin would break here. With this change a subproject does not need to declare the java plugin in advance (as is assumed by this plugin).